src/components/MoviesList/MoviesList.tsx

```diff
@@ -4,11 +4,14 @@ import { MovieCard } from '../MovieCard';
 
 interface Props {
   movies: Movie[];
+  visibleMovies: Movie[];
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
+export const MoviesList: React.FC<Props> = ({ movies, visibleMovies }) => (
   <div className="movies">
-    {movies.map(movie => (
+    {visibleMovies ? visibleMovies.map(movie => (
```

Comment:
visibleMovies  is always true, because it's an array , which is truth value. so simplify it